modeshape-jcr/src/main/resources/org/modeshape/jcr/repository-config-schema.json

```diff
@@ -28,6 +28,11 @@
             "default" : "10000",
             "description" : "The number of milliseconds to wait when a lock cannot be obtained on a node. In highly concurrent cases, this may be adjusted. Defaults to 10 seconds"
         },
+        "shellInterpreter" : {
```

Comment:
maybe "enableShell" would be a better name ? Or something else which suggests a boolean value